prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,348 @@
+
+The `user` object has a `gdpr` subobject that contains data about the user's GDPR preferences. 
+
+{: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| user.gdpr | Optional | `Object` |  Container object describing user GDPR preferences. |
+| gdpr.consentrequired | Optional | `Boolean` |  Indicates whether GDPR is in effect. |
+| gdpr.consentstring | Optional | `String` |  String that contains the data structure developed by the GDPR Consent Working Group under the auspices of IAB Europes. |
+
+### Price Range
+
+The `pricegranularity` subobject `range` describes the maximum price point for the price range and the increments to traverse that range. Visit our [price granualarity](/prebid-mobile/adops-price-granularity.html) overview for more details. 
+
+{: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| range | Optional | `Object` |  Container object price granualarity range. |
+| range.max | Optional | `Float` |   |
+| range.increment | Optional | `Float` |   |
+
+## Examples
+
+### Post Request
+
+ ```javascript
+ {
+  "storedrequestid": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+  "podconfig": {
+    "durationrangesec": [
+      15,
+      30
+    ],
+    "pods": [
+      {
+        "podid": 1,
+        "adpoddurationsec": 60,
+        "configid": "fba10607-0c12-43d1-ad07-b8a513bc75d6"
+      },
+      {
+        "podid": 2,
+        "adpoddurationsec": 60,
+        "configid": "8b452b41-2681-4a20-9086-6f16ffad7773"
+      }
+    ]
+  },
+  "site": {
+    "page": "http://www.foobar.com/1234.html"
+  },
+  "user": {
+    "yob": 1982,
+    "gender": "M",
+    "buyeruids": {
+      "appnexus": "unique_id_an"
+    },
+    "gdpr": {
+      "consentrequired": false,
+      "consentstring": ""
+    }
+  },
+  "device": {
+    "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/537.13 (KHTML, like Gecko) Version/5.1.7 Safari/534.57.2",
+    "ip": "123.145.167.10",
+    "devicetype": 1,
+    "ifa": "AA000DFE74168477C70D291f574D344790E0BB11"
+  },
+  "includebrandcategory": {
+    "primaryadserver": 1
+  },
+  "video": {
+    "w": 640,
+    "h": 480,
+    "mimes": [
+      "video/mp4"
+    ],
+    "protocols": [
+      2,
+      3
+    ]
+  },
+  "content": {
+    "episode": 6,
+    "title": "episodeName",
+    "series": "TvName",
+    "season": "season3",
+    "len": 900,
+    "livestream": 0
+  }
+}
+```
+
+### Post Response
+
+The POST response contains an array of `adpod` objects which represents the `adpods` in the request. Each value of the `adpod ` object (`podid:1`, `podid:2`, etc) contain the key-value targeting for those bids and optionally any errors encountered. 
```

Comment:
init cap the P in adPod/adPods?

Reply:
In the client side code adpod is all lowercase. I guess the discrepancy doesn't matter, cause if you are using one you are not using the other. Edited to reflect how server side writes it.